### DISCUSSION TITLE
Parallel processing capabilities

### DISCUSSION THREAD

**Discussion by IsotoCedex** [Universite-Gustave-Eiffel/NoiseModelling#645] [Q&A]
Hello,
It seems that some modeling software, not specifically for noise, implements parallel processing capabilities.
I wonder if this is the case with NoiseModelling.
In that case, would having an NVIDIA card with CUDA parallel processing capability improve the calculation speed of NoiseModelling?, or, would AMD's OpenCL and RDNA (Radeon DNA) technology be better?
Thank you.

**Comment by nicolas-f** [comment]
Hi,

NoiseModelling does not use Graphic Cards for computing noise maps. It is however use all available CPU threads.

 In another project we use a high performance computing solution ( SLURM ) in order to compute noise maps that cover all the country.

**Comment by pierromond** [comment]
Maybe one day : https://github.com/Universite-Gustave-Eiffel/NoiseModelling/issues/223
